crates/re_web_viewer_server/src/lib.rs

```diff
@@ -24,7 +24,7 @@ pub use sync::WebViewerServerHandle;
 
 pub const DEFAULT_WEB_VIEWER_SERVER_PORT: u16 = 9090;
 
-#[cfg(not(feature = "__ci"))]
+#[cfg(not(any(disable_web_viewer_server, feature = "__ci")))]
```

Comment:
`re_web_viewer_server` is a dependency of `re_sdk`, but `re_sdk` does not expose the `__ci` feature. Instead of adding one and propagating it all the way through, I added a custom `cfg` directive that will disable the web viewer server directly, and is set via `RUSTFLAGS`. This way it has no effect on anything else.

Reply:
This comment should be in the code